README.md

```diff
@@ -84,7 +84,7 @@ export default DS.Model.extend(ModelMixin, {
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  model: function(params) {
+  model(params) {
```

Comment:
This is ES6, right? I think that should use syntax most users are familiar with. The old syntax is closer to how ember's docs is written right now.